.travis.yml

```diff
@@ -12,13 +12,10 @@ branches:
     - travis
 
 rvm:
-  - 1.8.7
```

Comment:
We are Ruby 1.8 compatible in the low-level client, what's the reason for removing 1.8.7 from the Travis configuration?